model/trigger/task.go

```diff
@@ -440,15 +438,11 @@ func (t *taskTriggers) taskRuntimeChange(sub *event.Subscription) (*notification
 	}
 	thisTaskDuration := float64(t.task.FinishTime.Sub(t.task.StartTime))
 	prevTaskDuration := float64(lastGreen.FinishTime.Sub(lastGreen.StartTime))
-	ratio := thisTaskDuration / prevTaskDuration
-	if !util.IsFiniteNumericFloat(ratio) {
-		return nil, nil
-	}
-	percentChange := math.Abs(100*ratio - 100)
-	if percentChange < percent {
+	shouldNotify, percentChange := runtimeExceedsThreshold(percent, prevTaskDuration, thisTaskDuration)
+	if !shouldNotify {
 		return nil, nil
 	}
-	return t.generate(sub, fmt.Sprintf("changed in runtime by %f%% (over threshold of %f%%)", percentChange, percent))
+	return t.generate(sub, fmt.Sprintf("changed in runtime by %.1f%% (over threshold of %.1f%%)", percentChange, percent))
```

Comment:
Nit, but I'd advocate for using the original string representation of the float (percentString) for the second template parameter.

Reply:
done